dev/README.md

```diff
@@ -0,0 +1,16 @@
+# Nautilus-WebUI Docker-Compose
+
+**This folder is only for development purpose!**
+
+This docker-compose file creates a series of services for the Nautilus project, including database, backend, frontend, backend-tools, and backend-tests.
+
+## Usage
+
+1. Install Docker and Docker Compose on your machine.
+2. Clone the Nautilus Project repository.
+3. Navigate to the directory containing the `docker-compose.yml` file.
```

Comment:
We should probably advise to run from root of the repo with `docker compose -f dev/docker-compose.yml up`